app/views/catalog/_facet_pagination.html.erb

```diff
@@ -1,3 +1,13 @@
+    <% if @facet.index_pagination && params["facet.sort"] == 'index' %>
```

Comment:
I wonder if the index pagination controls should get pulled out to a separate partial.